### DISCUSSION TITLE
Not received the benifits even after 3 days of approval

### DISCUSSION THREAD

**Discussion by Tarunkumar314** [community/community#144849] [GitHub Education] [answered]
### Hi everyone,

I had applied for student developer pack and after three rejection it finally got approved but then it's showing that benefits will be available soon.. and when I searched it and it shows that it takes upto 3 days but it has been 3 days and benifits are still not there 

What should I do ??

**Comment by davevad93** [comment, marked as answer]
Hi @Tarunkumar314 ,

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @Tarunkumar314 was approved and your benefits are now available.`

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

For further reference you can look at these discussions: 
#132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268, #140287

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)
